src/main/java/com/shaft/api/RestActions.java

```diff
@@ -76,7 +77,7 @@ public static Response getLastResponse() {
     }
 
     public RestActions(String serviceURI) {
-        initializeSystemProperties(System.getProperty("apiConnectionTimeout") == null);
+        initializeSystemProperties();
```

Comment:
I changed the method removing this parameter because it'll never be null since the properties manager classes have default values for all properties. @MahmoudElSharkawy